python/lang/security/dangerous-globals-use.yaml

```diff
@@ -0,0 +1,31 @@
+rules:
+- id: dangerous-globals-use
+  patterns:
+  - pattern-either:
+    - pattern: globals().get(...)
+    - pattern: locals().get(...)
```

Comment:
`.get(...)` is a helper method for dictionary access - we'll also need to capture `globals()[...]` which does the same thing

Reply:
Like this: 

```
>>> locals()['__builtins__'].__dict__['abs'](-12)
12
```

Reply:
Otherwise, looks good! 👍

Reply:
Oh yeah, my bad! :+1:

Reply:
done!